src/main/webapp/app/exercises/shared/feedback/node/feedback-node.component.html

```diff
@@ -19,7 +19,9 @@ <h4 class="feedback-item__category">{{ feedbackItem.name }} {{ feedbackItem.titl
     <div class="feedback-group" [ngClass]="'bg-' + feedbackItemGroup.color" (click)="feedbackItemGroup.open = !feedbackItemGroup.open">
         <div class="feedback-group__left">
             <fa-icon class="me-2" [icon]="feedbackItemGroup.open ? faAngleUp : faAngleDown"></fa-icon>
-            <h4 class="feedback-item__category">{{ feedbackItemGroup.name | titlecase }} ({{ feedbackItemGroup.members.length }})</h4>
+            <h4 class="feedback-item__category">
+                {{ 'artemisApp.feedback.group.' + feedbackItemGroup.name | artemisTranslate | titlecase }} ({{ feedbackItemGroup.members.length }})
```

Comment:
Why do we need titlecase here, when we can already capitalize the translations in the file?

Reply:
Just in case it won't be capitalized some time

Reply:
```suggestion
                {{ 'artemisApp.feedback.group.' + feedbackItemGroup.name | artemisTranslate }} ({{ feedbackItemGroup.members.length }})
```